hero_selection.lua

```diff
@@ -766,7 +766,7 @@ function AllPickLogic()
 					(GetSelectedHeroName(id) == "" or GetSelectedHeroName(id) == nil))
 			then
 				if (randomTime == 0) then
-					randomTime = RandomInt(10, 12);
+					randomTime = RandomInt(1, 2);
```

Comment:
Maybe this is too fast?

Reply:
10 to 12 seconds would make players wait for around one minute. It could be better if bots can choose faster.